aws-s3-account-public-access-block/main.tf

```diff
@@ -0,0 +1,28 @@
+locals {
+  is_all  = var.restrict == "all"
+  is_new  = var.restrict == "new"
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>local.is_new is declared but not used